src/Microsoft.ML.Data/Transforms/InvertHashUtils.cs

```diff
@@ -390,9 +391,10 @@ private static void Save(IChannel ch, ModelSaveContext ctx, CodecFactory factory
             IValueCodec codec;
             var result = factory.TryGetCodec(new VectorType(TextType.Instance), out codec);
             ch.Assert(result);
-            ch.Assert(codec.Type.IsVector);
-            ch.Assert(codec.Type.VectorSize == 0);
-            ch.Assert(codec.Type.ItemType.RawType == typeof(ReadOnlyMemory<char>));
+            VectorType vectorType = codec.Type as VectorType;
+            ch.Assert(vectorType != null);
+            ch.Assert(vectorType.Size == 0);
+            ch.Assert(vectorType.ItemType.RawType == typeof(ReadOnlyMemory<char>));
```

Comment:
This *works* I guess, but would `ch.Assert(vectorType.ItemType == TextType.Instance);` be a bit more appropriate and in line with our patterns elsewhere? The expectation is that this is not just *any* `ColumnType` that happens to have read only memory of char as the underlying .NET type, but quite specifically the `TextType` itself, right? Which is a stronger statement. (In addition to being more concisely and clearly expressed.)

I know that wasn't how the original code was written -- indeed I'm pretty sure I'm the one that wrote it considering what file this is -- but, that was goofy of me. 😄 Not a big deal, just so long as you're fixing stuff anyway for @Ivanidzo4ka .

Reply:
Good call. Will fix.